curp/src/server/gc.rs

```diff
@@ -1,15 +1,13 @@
-use std::time::Duration;
+use std::{time::Duration, sync::Arc};
 
 use super::spec_pool::SpecPoolRef;
 use crate::{cmd::Command, server::cmd_board::CmdBoardRef};
-
-/// How often we should do gc
-const GC_INTERVAL: Duration = Duration::from_secs(20);
+use utils::config::CurpConfig;
 
 /// Run background GC tasks for Curp server
-pub(super) fn run_gc_tasks<C: Command + 'static>(cmd_board: CmdBoardRef<C>, spec: SpecPoolRef<C>) {
-    let _spec_pool_gc = tokio::spawn(gc_spec_pool(spec, GC_INTERVAL));
-    let _cmd_board_gc = tokio::spawn(gc_cmd_board(cmd_board, GC_INTERVAL));
+pub(super) fn run_gc_tasks<C: Command + 'static>(cmd_board: CmdBoardRef<C>, spec: SpecPoolRef<C>, curp_cfg: &Arc<CurpConfig>) {
```

Comment:
You can directly pass the `gc_interval` to `run_gc_tasks` rather than wrapping it by an `Arc`